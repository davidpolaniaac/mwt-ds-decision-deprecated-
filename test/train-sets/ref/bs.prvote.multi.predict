1.165888 0.997296 1.296013
0.091265 -0.024881 0.170573
2.711134 1.818539 2.858242
0.118992 -0.143870 0.338941
0.030651 -0.095287 0.146111
2.235398 1.812118 2.769007
-0.019054 -0.114057 0.118428
-0.094282 -0.182801 0.013014
2.854777 2.546088 3.000000
0.518617 0.381745 0.532782
0.272279 0.243442 0.290122
2.772975 0.514161 2.873086
0.054138 0.010703 0.138818
0.086204 -0.102617 0.241717
1.900103 1.773769 2.113003
0.921255 0.879776 0.972381
1.081151 0.881054 1.215401
2.245001 2.035578 2.444676
0.157608 0.090615 0.678673
0.317768 0.195989 0.416920
2.158592 0.572330 2.175719
1.135644 0.948689 1.426823
-0.039610 -0.204115 0.754271
2.195246 1.976621 2.535813
-0.139204 -0.228134 -0.001704
0.187221 0.026379 0.416575
2.766444 2.539795 3.000000
0.127871 -0.081224 0.289972
0.867238 0.583283 1.102605
0.726878 0.657689 0.770682
0.917494 0.888209 0.976421
0.211885 0.044036 0.394508
2.390648 2.322996 2.994010
0.159319 0.045135 0.300119
1.002115 0.958525 1.093620
2.896398 2.827745 3.000000
0.945461 0.747264 1.063062
-0.067568 -0.233451 0.062793
2.116748 1.754919 2.864041
1.353323 1.280505 1.389261
0.010613 -0.054453 0.070220
2.365053 2.084350 2.498975
-0.045694 -0.150979 0.059899
0.111073 0.024889 0.836689
2.181469 1.708614 2.601858
0.091936 -0.097144 0.498147
0.605386 0.022822 0.647841
2.341661 2.233139 2.637181
0.901337 0.836292 0.981192
0.626944 0.543057 0.721317
2.193523 1.957688 2.779601
0.889248 0.805914 0.989266
2.782458 2.725823 2.850785
0.072403 -0.140018 0.444593
1.323045 1.091665 1.485385
2.796370 2.198494 2.856077
-0.150043 -0.227155 -0.064318
2.135840 1.818663 2.761198
0.839874 0.709630 0.989899
0.690396 0.638766 0.754106
-0.301880 -0.606652 0.068781
-0.049124 -0.191876 0.093611
0.886763 0.792914 0.967395
-0.131103 -0.181684 -0.027351
-0.724531 -0.892949 -0.585002
-0.139397 -0.201341 -0.095313
-0.039411 -0.128911 0.139969
-0.995490 -1.000000 0.483943
0.064290 -0.090407 0.242410
0.850627 0.764400 0.931225
-0.580725 -0.604451 -0.364373
1.044828 0.941816 1.142960
0.881977 0.721201 1.065755
0.191367 0.026044 0.368995
0.736443 0.606240 0.998738
1.001133 0.954040 1.026910
-0.040143 -0.137300 0.106709
-0.078780 -0.170735 0.030993
0.018575 -0.036332 0.069993
0.967803 0.834524 1.054736
-0.036761 -0.147490 0.080111
-0.018494 -0.092213 0.045656
1.076620 0.983203 1.133457
-0.888092 -0.991592 -0.761992
-0.063347 -0.164116 0.017370
-0.035214 -0.090992 0.067626
1.035496 0.936973 1.137832
0.972349 0.911019 1.096976
-0.299250 -0.896316 -0.166576
0.675929 0.598708 0.781356
0.034932 -0.017629 0.119009
0.956413 0.933704 0.984708
1.080908 0.859511 1.274853
-0.015273 -0.055805 0.013013
-0.180945 -0.684304 -0.145759
-0.022553 -0.130465 0.057897
1.016581 0.903859 1.075045
0.076344 0.006028 0.132119
1.124847 1.070429 1.178548
0.665210 0.557164 0.740026
0.127635 0.012004 0.306633
0.939810 0.813142 1.039797
0.307641 0.219353 1.045170
-0.724036 -0.918504 -0.120672
0.118309 -0.017080 0.275518
1.034220 0.949006 1.107581
-0.019488 -0.072708 0.047928
0.009654 -0.081827 0.117147
2.805850 2.162519 2.963920
0.692761 0.617064 0.745980
1.002576 0.954738 1.051530
1.826252 0.837229 2.520384
0.002416 -0.078817 0.063057
1.067481 0.871075 1.264038
0.261793 0.118992 0.615175
-0.011782 -0.085065 0.025072
2.006676 0.431598 2.093800
1.072785 0.941992 1.273427
0.853569 0.764991 0.966257
0.434191 0.409039 0.572215
-0.047874 -0.081031 0.012337
1.012655 0.986233 1.035113
2.668044 1.696929 2.823445
1.023344 0.922767 1.197439
1.077659 0.993362 1.158937
0.758295 0.356116 0.861828
-0.008981 -0.075009 0.052994
0.958572 0.760490 1.083998
-0.005249 -0.086802 0.062083
0.689489 0.561948 0.814420
0.161110 0.104884 0.209097
0.951251 0.849945 1.048296
0.140934 0.106123 0.168837
0.974733 0.914669 1.033538
2.840775 1.930148 2.896898
-0.021813 -0.083969 0.125262
0.958498 0.876787 1.080578
0.947131 0.935528 0.970666
1.261632 1.140657 1.348179
0.369828 0.281447 0.419381
0.141834 0.035235 0.396061
0.016443 -0.093786 0.196600
1.125101 1.096046 1.159018
2.098608 1.898286 2.456885
0.858513 0.831951 0.905605
0.816551 0.650382 0.916158
0.981262 0.749956 1.184740
2.288061 1.960806 2.448737
-0.119189 -0.147815 0.580457
0.967754 0.882084 1.055152
0.807915 0.606088 0.920796
2.925095 2.855120 3.000000
0.838593 0.561854 0.946930
0.197615 0.148620 0.228379
2.825346 2.760775 2.901104
0.834976 0.675480 0.927743
0.937966 0.778486 1.035339
2.720611 2.323100 2.851998
0.399221 0.329307 0.665108
0.847648 0.799572 0.933874
0.196500 -0.008269 0.375510
-0.103772 -0.328219 0.009213
-0.160399 -0.229804 -0.116273
1.021716 -0.419678 1.121495
0.032702 -0.032790 0.140288
0.986105 0.951496 1.019486
0.915283 0.728512 1.002046
0.124798 0.090012 0.720025
-0.939583 -1.000000 -0.758333
1.043210 0.900743 1.112997
1.005975 0.910205 1.078144
0.029579 -0.029108 0.167298
-0.049064 -0.169489 0.030176
0.846442 0.378798 0.906329
0.175336 0.022676 0.750413
-0.061051 -0.126447 -0.016095
-0.873418 -0.906557 0.160036
0.977896 0.939241 1.008837
0.977534 0.950832 0.995436
0.697497 0.659374 0.733258
1.061880 0.929425 1.213291
0.213046 0.124843 0.279854
0.887127 0.787096 0.995988
-0.025487 -0.067783 0.037388
-0.027442 -0.098959 0.035617
-0.862329 -0.983385 -0.636670
0.920245 0.775490 0.982473
0.062446 0.001356 0.148851
-0.019664 -0.047406 0.005275
0.882899 0.474872 1.044118
0.964368 0.890747 1.018061
0.042732 -0.007758 0.078391
-0.001784 -0.003498 0.000124
-0.014917 -0.053353 -0.000365
0.040967 0.009760 0.075245
0.981875 0.935274 0.999042
0.997410 0.910967 1.103930
0.028158 -0.030937 0.145855
0.000531 0.000072 0.680008
0.759627 0.448572 0.873944
