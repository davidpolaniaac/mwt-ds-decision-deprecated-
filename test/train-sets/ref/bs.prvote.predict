0.478539 0.458390 1.000000
0.566167 0.452659 0.619416
0.406805 0.310507 0.456509
0.251622 0.173337 0.336450
0.322963 0.248373 0.432724
0.983781 0.935124 1.000000
0.336023 0.283262 0.428032
0.192092 0.045960 0.367021
0.412438 0.355603 0.473136
0.987273 0.949092 1.000000
0.299938 0.191055 0.379315
0.383079 0.324045 0.538324
0.282031 0.191435 0.419371
0.338635 0.310091 0.535169
0.944859 0.902809 0.994895
0.690543 0.592456 0.885632
0.944518 0.860310 1.000000
0.172964 0.057171 0.353277
0.329856 0.190818 0.644261
0.121756 0.045328 0.169037
0.262164 0.230530 0.923944
0.850630 0.611789 1.000000
0.193337 0.070341 0.310366
0.991279 0.489735 1.000000
0.063488 0.000000 0.151716
0.269885 0.138823 0.475542
0.281528 0.224301 0.449844
0.156322 0.044406 0.248419
0.854318 0.468885 0.996748
0.289275 0.243104 0.542972
0.864170 0.605581 1.000000
0.326018 0.138691 0.428626
0.164169 0.122979 0.198036
0.354337 0.272718 0.450172
0.851262 0.685431 0.997519
0.292653 0.175658 0.570998
0.843352 0.654157 1.000000
0.169977 0.099213 0.276944
0.895102 0.602038 1.000000
1.000000 1.000000 1.000000
0.257839 0.109665 0.408097
0.890388 0.826258 0.946926
0.052490 0.011032 0.125593
0.292537 0.147675 0.375115
0.169259 0.034709 0.309596
0.129500 0.083988 0.181821
0.257917 0.119488 0.701049
0.056381 0.043627 0.557042
0.937568 0.290648 0.961409
0.256776 0.222167 0.287373
0.959284 0.428921 1.000000
1.000000 0.368642 1.000000
0.127635 0.000000 0.401585
0.232054 0.092099 0.481334
0.992366 0.969464 1.000000
0.866602 0.222806 1.000000
0.193469 0.119060 0.317708
0.224850 0.069827 0.380972
0.412085 0.371972 0.885179
0.188716 0.067895 0.437237
0.887909 0.570298 1.000000
0.108563 0.016143 0.348595
0.714218 0.597393 0.885829
0.124156 0.066054 0.201452
0.905294 0.621175 1.000000
0.155813 0.093998 0.313378
0.331615 0.069193 0.460195
0.122725 0.095883 0.806661
0.114768 0.045854 0.196326
0.632674 0.566345 0.797911
0.158623 0.056481 0.425846
0.677401 0.539467 0.902148
0.325638 0.283821 0.976362
0.048461 0.000000 0.127378
0.890640 0.726537 1.000000
1.000000 0.459947 1.000000
0.164979 0.029138 0.371215
0.355033 0.187811 0.601652
0.100235 0.057713 0.119670
0.224388 0.059909 0.347789
0.048057 0.000000 0.192228
0.214221 0.141604 0.400031
0.913158 0.787474 1.000000
0.286814 0.102555 0.489686
0.173353 0.150986 0.222639
0.214987 0.117383 0.415650
0.364587 0.345910 1.000000
0.835789 0.664915 1.000000
0.943070 0.772280 1.000000
0.093061 0.000000 0.300062
0.142314 0.053952 0.209100
0.841398 0.534163 0.951390
1.000000 1.000000 1.000000
0.130773 0.072309 0.175301
0.798578 0.681810 1.000000
0.053998 0.000000 0.213567
0.768004 0.504172 1.000000
0.170715 0.027735 0.319510
0.921924 0.384518 0.950059
0.863378 0.533635 1.000000
0.132375 0.000000 0.401651
0.710231 0.534494 0.939360
0.040689 0.000000 0.104174
0.879732 0.493532 1.000000
0.083491 0.000000 0.147888
0.987010 0.963237 1.000000
0.056283 0.000000 0.143924
0.193988 0.048358 0.415193
0.166880 0.133686 0.228804
0.968765 0.917059 1.000000
0.837322 0.665323 1.000000
0.618216 0.154025 0.628948
0.106209 0.000000 0.408536
0.351225 0.317683 1.000000
0.056086 0.000000 0.509912
0.049482 0.000000 0.506964
0.886553 0.323823 0.987185
0.877494 0.452777 1.000000
0.403805 0.327837 1.000000
0.160269 0.001862 0.496232
0.119045 0.018962 0.388772
0.937201 0.927328 0.949857
0.125252 0.000000 0.404609
0.952529 0.453135 1.000000
0.699265 0.495457 0.991451
0.801212 0.369999 0.885172
0.133208 0.000000 0.405252
0.814467 0.449116 0.886408
0.158933 0.000000 0.400786
0.804983 0.414330 0.945832
0.043934 0.000000 0.117271
0.891290 0.733348 1.000000
0.137828 0.037800 0.243426
0.261271 0.237421 0.939561
0.103741 0.000000 0.367407
0.211073 0.026962 0.300478
0.855485 0.496068 1.000000
0.403767 0.350595 0.995408
0.987643 0.954148 1.000000
0.251342 0.029820 0.379896
0.218393 0.002710 0.456161
0.074298 0.000000 0.174533
0.791924 0.683823 0.909649
0.910751 0.823318 0.990685
0.944804 0.408713 0.985697
0.811008 0.611208 0.920679
0.921439 0.488054 0.945687
0.965981 0.892140 1.000000
0.080854 0.000000 0.323415
0.895920 0.309824 0.909166
0.952046 0.908086 1.000000
0.223357 0.215659 0.913834
0.832780 0.622499 0.926225
0.091201 0.000000 0.364806
0.583699 0.000000 0.663357
0.813661 0.528311 0.920641
0.974728 0.937749 1.000000
0.016536 0.000000 0.061637
0.962409 0.849636 1.000000
0.004912 0.000000 0.621811
0.839594 0.548149 0.956069
0.053547 0.000000 0.203684
0.179329 0.040461 0.379248
0.773867 0.678339 0.936412
0.095439 0.000532 0.331340
0.922309 0.409029 0.923825
0.976775 0.933654 1.000000
0.040832 0.000000 0.086990
0.457036 0.417787 0.997241
0.920089 0.705726 1.000000
0.712788 0.584154 0.913484
0.249767 0.029493 0.334615
0.118174 0.000000 0.273973
0.380593 0.314658 0.826037
0.117311 0.045142 0.200166
0.254937 0.031250 0.597997
0.150506 0.025126 0.275946
0.886845 0.774050 0.971336
0.907234 0.677971 1.000000
0.835232 0.733083 0.959718
0.814622 0.448380 0.996360
0.132508 0.083232 0.174466
0.892350 0.425506 0.908774
0.066416 0.000000 0.248872
0.170242 0.068516 0.384858
0.022225 0.000000 0.513376
0.840112 0.627244 1.000000
0.033030 0.000000 0.558645
0.211386 0.036497 0.363344
0.811269 0.671142 0.994183
0.792029 0.661582 0.920168
0.247624 0.075627 0.462856
0.012373 0.000000 0.035419
0.160132 0.000000 0.297109
0.188652 0.063500 0.297037
0.993022 0.982524 1.000000
0.468484 0.450480 0.962684
0.007095 0.000000 0.019079
0.219083 0.018797 0.458421
0.771669 0.532585 1.000000
