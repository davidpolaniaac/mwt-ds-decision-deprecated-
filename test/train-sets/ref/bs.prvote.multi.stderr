only testing
Num weight bits = 18
learning rate = 10
initial_t = 1
power_t = 0.5
predictions = bs.prvote.multi.predict
using no cache
Reading datafile = train-sets/0001.multi.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
0.027519   0.027519            1         1.0   1.0000   1.1659       51
0.017924   0.008329            2         2.0   0.0000   0.0913      104
0.033363   0.048801            4         4.0   0.0000   0.1190      135
0.091032   0.148702            8         8.0   0.0000  -0.0943      146
0.145818   0.200604           16        16.0   1.0000   0.9213       24
0.305401   0.464983           32        32.0   0.0000   0.2119       32
0.250688   0.195975           64        64.0   0.0000  -0.1311       61
0.220722   0.190757          128       128.0   1.0000   0.9586      106

finished run
number of examples per pass = 200
passes used = 1
weighted example sum = 200
weighted label sum = 0
average loss = 0.246666
best constant = -0.00502513
total feature number = 15482
